MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -621,40 +1041,77 @@ int main(int argc, char * argv[])
                         std::wcout << "Successfully unmounted the CIM with volume id: " << cli.GetVolumeId() << std::endl;
                     }
 
+                    // Telemetry : Workflow Log
+                    QueryPerformanceCounter(&msixMgrLoad_EndCounter);
+                    workflowElapsedTime = msixmgr::CalcWorkflowElapsedTime(msixMgrLoad_StartCounter, msixMgrLoad_EndCounter, msixMgrLoad_Frequency);
+                    msixmgr::TraceLogWorkflow(workflowId.c_str(), cli.GetOperationTypeAsString().c_str(), true, workflowElapsedTime, L"", L"");
+
                     std::wcout << std::endl;
                 }
             }
             else if (fileType == WVDFileType::VHD || fileType == WVDFileType::VHDX)
             {
                 if (cli.GetMountImagePath().empty())
                 {
+                    errorCode = msixmgr::GetErrorCodeFromHRESULT(E_INVALIDARG);
+                    errorDesc = L"Path of the VHD(X) image to be unmounted not given. HRESULT " + errorCode + L".";
+
                     std::wcout << std::endl;

```

Comment:
Same as above

Reply:
As discussed, all the Telemetry variables will have the same Error Message as the actual Console statements. The change for outputting the Error Message on Console from a variable will be done as part of Error Message Improvement Task that will be taken later.